### PR TITLE
Detect Roman's extension on install

### PR DESCRIPTION
**Pull request by oldergod**
Contributes to #113

### CODE REVIEW

File: frontend/chrome-extension/manifest.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "keyword": "$keyword"
   },
   "permissions": [
+    "management",
```
Comment (oldergod) [inline frontend/chrome-extension/manifest.json]: That sucks

File: frontend/chrome-extension/background/src/main/kotlin/com/jakewharton/sdksearch/background/Background.kt [hunk 2]
```diff
@@ -33,4 +35,14 @@ fun main(vararg args: String) {
 
     SearchUiBinder(presenter.events, Chrome, config.dacUrl).bindTo(presenter)
   }
+
+  Chrome.runtime.onInstalled.addListener { details ->
+    if (details["reason"] == "install") {
+      Chrome.management.get(THE_OTHER_SDK_SEARCH_ID) {
```
Comment (JakeWharton) [inline frontend/chrome-extension/background/src/main/kotlin/com/jakewharton/sdksearch/background/Background.kt]: Oh I didn't even know this API existed!

I was going to try using https://developer.chrome.com/extensions/runtime#method-connect, but I don't know if it works across extensions.

Comment (oldergod) [reply frontend/chrome-extension/background/src/main/kotlin/com/jakewharton/sdksearch/background/Background.kt]: I'll give it a try, it that works I could get rid of the `management` permission

Comment (JakeWharton) [reply frontend/chrome-extension/background/src/main/kotlin/com/jakewharton/sdksearch/background/Background.kt]: It seems to always return a `Port` instance even if you type in a garbage extension ID. I don't see a way to determine if the `Port` is actually connected or not.

Comment (oldergod) [reply frontend/chrome-extension/background/src/main/kotlin/com/jakewharton/sdksearch/background/Background.kt]: no luck

**Comment by JakeWharton** [comment]
Thanks for the effort but closing this and the issue. Oh well...

**Pull request closed without merging**